packages/app-builder-lib/src/configuration.ts

```diff
@@ -129,10 +129,10 @@ export interface CommonConfiguration {
    */
   readonly npmRebuild?: boolean
   /**
-   * Use `legacy` app-builder binary for installing native dependencies, or `@electron/rebuild` in `sequential` or `parallel` compilation modes.
+   * Use `@electron/rebuild` in `sequential` or `parallel` compilation modes.
    * @default sequential
    */
-  readonly nativeRebuilder?: "legacy" | "sequential" | "parallel" | null
+  readonly nativeRebuilder?: "sequential" | "parallel" | null
```

Comment:
This requires this PR to be a `major` semver changeset. Great PR to prep for when we want to do a Node 22 engine upgrade to match `electron` upstream